lib/Screens/Main.dart

```diff
@@ -36,16 +47,14 @@ class _MainNavigatorState extends  State<MainNavigator> {
     });
   }
 
-  void _setScreenName(String name) {
-    setState(() {
-      _screenName = name;
-    });
+  void _setScreenName(String screenName) {
+    screenNav.changeScreen(screenName);
```

Comment:
It might be useful to enumerate screen name at some point not needed now

Reply:
Could be hard with the translations

Reply:
oh yeah, i had that in the back of my mind - we will follow the convention you establish.